package.json

```diff
@@ -39,6 +39,7 @@
     "sinon": "1.10.3",
     "sinon-chai": "2.5.0",
     "bower": "1.3.12",
-    "grunt-sass": "0.17.0"
+    "grunt-sass": "0.17.0",
+    "underscore": "1.8.3"
   }
```

Comment:
in the ci script we simply move the index file :) it is a total hack but it works :)